docs/PowerShell.md

```diff
@@ -1,10 +1,56 @@
 # ParquetSharp in PowerShell
 
+### Installation
+
 It's possible to use ParquetSharp from PowerShell.
-You can install ParquetSharp with the [NuGet command line interface](https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference),
-then use `Add-Type` to load `ParquetSharp.dll`.
-However, you must ensure that the appropriate `ParquetSharpNative.dll` for your architecture and OS can be loaded as required,
-either by putting it somewhere in your `PATH` or in the same directory as `ParquetSharp.dll`.
-For examples of how to use ParquetSharp from PowerShell,
-see [these scripts from Apteco](https://github.com/Apteco/HelperScripts/tree/master/scripts/parquet).
+You can install ParquetSharp with the [NuGet command line interface](https://docs.microsoft.com/en-us/nuget/reference/nuget-exe-cli-reference).  
+
+First, make sure `nuget.exe` is in your `PATH`, or in the current directory, then run the following to install the latest version of ParquetSharp into a new directory called `lib`:
```

Comment:
These instructions are presented as if this is the one way of using ParquetSharp from PowerShell, but there's no requirement that the dlls are in a directory named "bin" for example. I think it's worth clarifying that this is just one possible approach, and we should keep the existing text that says that the main requirement is that `ParquetSharpNative.dll` needs to be on the `PATH` or in the same directory as `ParquetSharp.dll`.